frontend/src/app/users/users.component.ts

```diff
@@ -12,9 +15,22 @@ export class UsersComponent implements OnInit {
 
   users: Array<UserModel>;
 
-  constructor(private route: ActivatedRoute) { }
+  constructor(private route: ActivatedRoute,
+              private confirmService: ConfirmService,
+              private userService: UserService) { }
 
   ngOnInit() {
     this.users = sortBy<UserModel>(this.route.snapshot.data['users'], u => u.login);
   }
+
+  delete(user: UserModel) {
+    this.confirmService.confirm({message: `Voulez-vous vraiment supprimer l\'utilisateur ${user.login}\u00A0?`})
+      .switchMap(() => this.userService.delete(user.id))
+      .switchMap(() => this.userService.list())
+      .subscribe(users => this.users = sortBy(users, u => u.login), () => {});
```

Comment:
Without it, the exception handler or something like that detects an unhandled error if the user doesn't confirm, which prints an error in the console and/or in the tests (or maybe makes the test fail, IIRC)